config/datatypes_conf.xml.sample

```diff
@@ -51,7 +51,7 @@
       <converter file="bed_gff_or_vcf_to_bigwig_converter.xml" target_datatype="bigwig"/>
     </datatype>
     <!-- MSI added Datatypes -->
-    <datatype extension="csv" type="galaxy.datatypes.tabular:Tabular" subclass="True" display_in_upload="true" /> <!-- FIXME: csv is 'tabular'ized data, but not 'tab-delimited'; the class used here is intended for 'tab-delimited' -->
+    <datatype extension="csv" type="galaxy.datatypes.tabular:CSV" subclass="True" display_in_upload="true" />
```

Comment:
I think all (if not most) tabular tools have been converted to work with variable separator, my idea is that the same tools should work with **any** tabular dialect. Maybe more work has to be done on this, but I have yet to see a project where CSV, TSV, ... are treated as different types, e.g. *shell, awk, shell, R, ...* all work the same on the same dataset with any specified separator. I think the sources should use more of the [csv](https://docs.python.org/2/library/csv.html) package that's in Python since 2.3 instead of handwritten parsing and all should be fine.